### DISCUSSION TITLE
hachette@Poland --> Opowiadajki

### DISCUSSION THREAD

**Discussion by pokryva** [oyooyo/audiocube#3] [General]
Hi @oyooyo!
Just letting you know - the "hachette" version in Poland is sold as Opowiadajki (https://opowiadajki.pl/).

**Comment by oyooyo** [comment]
Thank you very much for telling me, I am always glad finding out about a new localized versions of this device!

I'd actually like to create separate sub-pages for the various national "hachette" versions. Any chance you could create a page for the polish version, similar to this page for the german version?
https://oyooyo.github.io/audiocube/devices/hachette/DE/

Basically, it's just about creating a table that lists the tales that are part of the local hachette collection sold in Poland. Most of the columns in that table are rather pointless and will probably be removed, the only important pieces of information are the names of the fairytales in polish, and the filename on the Micro SD card.

I already created an empty template for the polish version here:
https://github.com/oyooyo/audiocube/blob/master/docs/devices/hachette/PL/index.md

**Reply by pokryva** [reply]
Hi, I sure will do my best and be more than happy to contribute! 
We've only had 5 fairy tales so far here in Poland (meaning - as a subscriber I've only received 5 figures by now), but the first SD card has 38 smp files - so I should be able to complete the list per your template.

**Comment by oyooyo** [comment]
It's quite similar over here. Only a minority of the fairytale figures have been released so far, but since the SD card already contains all fairytales that are yet to come, it was possible to publish a complete list.
Would be great if you could create such a list for the polish collection!

**Comment by cyrmacgtm** [comment]
Hey, i have issue number 31 with the second SD card, and there are all SMP files with all fairytales. These are this list with corresponding filenames: 
_(note that order of fairytales in sd card is different than issues by number)_
| filename | title |
| --- | --- |
| `T0001.smp` | Czerwony kapturek |
| `T0002.smp` | Trzy świnki |
| `T0003.smp` | Pinokio |
| `T0004.smp` | Kot w butach |
| `T0005.smp` | Jaś i Małgosia |
| `T0006.smp` | Dzielny ołowiany żołnierzyk |
| `T0007.smp` | Śpiąca królewna |
| `T0008.smp` | Brzydkie Kaczątko |
| `T0009.smp` | Mała Syrenka |
| `T0010.smp` | Księga dżungli |
| `T0011.smp` | Królewna Śnieżka i siedmiu krasnoludków |
| `T0012.smp` | Opowieść wigilijna |
| `T0013.smp` | O dzielnym krawczyku |
| `T0014.smp` | Dziadek do orzechów |
| `T0015.smp` | Złotowłosa i trzy niedźwiadki |
| `T0016.smp` | Flecista z Hameln |
| `T0017.smp` | Kopciuszek |
| `T0018.smp` | Nowe szaty cesarza |
| `T0019.smp` | Alicja w Krainie Czarów |
| `T0020.smp` | Czarnoksiężnik z Krainy Oz |
| `T0021.smp` | Zarozumiała Myszka |
| `T0022.smp` | Opowieść o zbieraczu bambusu |
| `T0023.smp` | Muzykanci z Bremy |
| `T0024.smp` | Księżniczka na ziarnku grochu |
| `T0025.smp` | Robin Hood |
| `T0026.smp` | Konik polny i mrówki |
| `T0027.smp` | Sindbad żeglarz |
| `T0028.smp` | Roszpunka |
| `T0029.smp` | Podróże Guliwera |
| `T0030.smp` | Mleczarka |
| `T0031.smp` | Alibaba i 40 rozbójników |
| `T0032.smp` | Tomcio Paluszek |
| `T0033.smp` | Żółw i zając |
| `T0034.smp` | Piękna i Bestia |
| `T0035.smp` | Książę i żebrak |
| `T0036.smp` | Słowik |
| `T0037.smp` | Jaś i magiczna fasola |
| `T0038.smp` | Lampa Aladyna |
| `T0039.smp` | Rumpelsztyk |
| `T0040.smp` | O wilku i siedmiu koźlątkach |
| `T0041.smp` | Dzwonnik z Notre Dame |
| `T0042.smp` | Królowa Śniegu |
| `T0043.smp` | Bajka o jednym takim co wyruszył w świat żeby strach poznać |
| `T0044.smp` | Bajki o krasnoludkach |
| `T0045.smp` | Moby Dick |
| `T0046.smp` | Piotruś Pan |
| `T0047.smp` | Heidi |
| `T0048.smp` | Upiór rodu Canterville'ów |
| `T0049.smp` | Lis i winogrona |
| `T0050.smp` | Król Artur i Rycerze Okrągłego Stołu |
| `T0051.smp` | Mysz z miasta i mysz ze wsi |
| `T0052.smp` | Atena i Arachne |
| `T0053.smp` | Kura znosząca złote jajka |
| `T0054.smp` | Legenda o Pegazie |
| `T0055.smp` | Dziewczynka z zapałkami |
| `T0056.smp` | Wilhelm Tell |
| `T0057.smp` | Szczur i lew |
| `T0058.smp` | Tarō Urashima |
| `T0059.smp` | Dzikie Łabędzie |
| `T0060.smp` | Piotruś i Wilk |
